README.md

```diff
@@ -130,12 +132,10 @@ py -3 -m venv venv
 
 ```bash
 python -m pip install -U pip
-python -m pip install -e .[dev]
-python -m gamatrix
+python -m pip install .
```

Comment:
Since this is for normal running of the project, not dev, I don't think we want editable mode or to specify `.[dev]`.